src/convertToObject.js

```diff
@@ -13,6 +13,30 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const resultObject = {};
+  const resultArray = [];
+
+  const array = sourceString.split(';');
+
+  array.forEach(element => {
+    const newElement = element.split(':');
+
+    resultArray.push(newElement);
+  });
+
+  const filterArray = resultArray.filter(item => item.length > 1);
+
+  filterArray.forEach(element => {
```

Comment:
```suggestion
  filterArray.forEach([key, value] => {
```
Тут можно использовать деструктуризацию